frontend/src/app/components/MainInstallButton/index.js

```diff
@@ -29,15 +29,15 @@ export default class MainInstallButton extends React.Component {
   enableExperimentWrapped() {
     const { experiment, enableExperiment, postInstallCallback } = this.props;
 
-    this.setState({ isInstalling: true }, () => enableExperiment(experiment)
+    this.setState({ isInstalling: true });
+    enableExperiment(experiment)
       .then(() => {
         if (postInstallCallback) {
           postInstallCallback();
-          this.setState({isInstalling: false});
```

Comment:
This triggered a warning by react saying the component had already been unmounted